README.md

```diff
@@ -129,15 +129,15 @@ file):
 
 ```
 docker build -t get_together .
-docker run -e "DEBUG_MODE=True" -e "SECRET_KEY=xxxxx" -e "ALLOWED_HOSTS=localhost,127.0.0.1" -d --name get_together -p 8000:8000 get_together
+docker run -e "DEBUG_MODE=True" -e "SECRET_KEY=xxxxx" -e "ALLOWED_HOSTS=localhost,127.0.0.1,0.0.0.0" -d --name get_together -p 8000:8000 get_together
```

Comment:
I'm not familiar with this requirement for docker, so I'll leave this to @mhall119

Reply:
I'm not sure this will do what you want it to do. ALLOWED_HOSTS is used by Django to match against the HTTP Host header, it's not the network address that Django will listen on. 

What you would want to do instead is pass whatever the actual IP address is of your docker container, or a hostname that resolves to it, that you will be putting in your browser to connect to the site.

Reply:
Reverted the `ALLOWED_HOSTS` change

Currently, I need to rebuild the Docker image everytime I change code locally. Would it be okay if I mounted the project directory to `/home/python/`?

Reply:
The docker image is meant to run self-contained code. Are you using it for a live development environment?

Reply:
I've always used Docker Compose to set up a local development environment.

Reply:
Perhaps we should provide a separate Dockerfile for this purpose. I'm going to go ahead and approve this PR as it is not, but feel free to create the new Dockerfile in a new PR.

Reply:
SG. This is why I asked about how this project is deployed. Don't want to break something that is already working.